tests/src/python/test_qgsserver.py

```diff
@@ -1164,18 +1319,14 @@ def _img_diff_error(self, response, headers, image, max_diff=10, max_size_diff=Q
         test, report = self._img_diff(response, image, max_diff, max_size_diff)
 
         with open(os.path.join(tempfile.gettempdir(), image + "_result.png"), "rb") as rendered_file:
-            encoded_rendered_file = base64.b64encode(rendered_file.read())
```

Comment:
Do not hesitate to correct me if I am wrong, but I think I just removed the binary dump in log. Images are still saved and available on CDash. 

And I removed it because when my tests was failing (I had to build some masks), Travis was failing because of the limit size exceeding 4MB for logs (due to the binary dump).

What do you think?